### PR TITLE
klayout: Update to 0.28.13

### PR DESCRIPTION
**Pull request by hashhsah**
**Summary**

There are dozens of new features and enhancements between 0.28.9 and 0.28.13. A complete list of changes can be found at https://www.klayout.de/development.html

**Test Plan**

- Open a few GDS and OAS files (kB to GB in size) and browse the layout.
- Open Help->About and check it ruby/python bindings are present in the `Build options` and ~20 .so files are present in `Binary Extensions`.
- Start python3, import `klayout` and `pya` modules.

**Checklist**

- [ ] Package was built and tested against unstable

### CODE REVIEW

File: packages/k/klayout/package.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ builddeps  :
     - pkgconfig(Qt5XmlPatterns)
     - pkgconfig(python3)
     - pkgconfig(ruby-3.2)
+    - pkgconfig(libgit2)
```
Comment (ReillyBrogan) [inline packages/k/klayout/package.yml:20]: `pkgconfig` dependencies are ordered alphabetically with capitalized dependency names going first and then lower-case once going after those. So `pkgconfig(libgit2)` should be moved up and placed after `pkgconfig(Qt5XmlPatterns)` and before `pkgconfig(python3)`.

File: packages/y/yosys/package.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 name       : yosys
-version    : '0.23'
-release    : 18
+version    : '0.35'
```
Comment (ReillyBrogan) [inline packages/y/yosys/package.yml:2]: This yosys update seems unrelated to the `klayout` update? Did you mean to include it or was it included accidentally? Neither package seems to have a run or builddep on the other, and while I'm not familiar enough with what they're doing they seem to no be needed for each other? If this was included accidentally, could you please move the yosys update to a separate PR?

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
Hi @hashhsah , it seems that reviewers have kept skipping over this PR in the review queue and for that I am deeply sorry. It should not have taken us this long to get around to reviewing this, we are very appreciative that you have taken the time to update these packages! 

Besides the two comments I left on the files I'd also like to mention that the commit order is incorrect. Package PRs should not have merge commits in the commit log, you'll need to rebase this PR so that only the commits that are actually changing packages are present in the commits to be added. If you need any help with how to do that please feel free to hop onto our Matrix and we'll be more than happy to help, or you can ask here and we can walk you through it as well!

**Comment by ReillyBrogan** [comment]
Oh I just noticed that this PR has a conflict with main. To fix that you'll need to rebase the commit, bump it and rebuild it so that the release number in the updated package.yml is 1 greater than what is currently in the main branch.

**Comment by ReillyBrogan** [comment]
Did you mean to close this?

**Comment by hashhsah** [comment]
sorry about the late reply, have been busy with work. Meanwhile klayout 0.28.14 has been released. I took note of the comments above and created a new PR #1153.

**Pull request closed without merging**